### ISSUE TITLE
Bug: File names are reset when adding photos to new remote album

### ISSUE THREAD

**Issue by marcus-k** [scubajeff/lespas#105]
**Describe the bug**
When adding pictures to a new remote album from the Les Pas app, the file names are not preserved when they are uploaded to the Nextcloud server. This did not used to happen before; my old uploaded photos have their file names as they are on my phone (i.e. yyyymmdd_hhmmss.jpg). However, any new photos I add to a new remote album get renamed to some random number that increments up for that album (see screenshot). Perhaps a new update changed this behaviour, I am not sure. I would like the file names to be preserved like prior photos I have uploaded.

**To Reproduce**
1. Click the '+' icon to create select photos
2. Create a new remote album for those photos
3. Observe the uploading where all the file names are renamed to the random numbers.

**Expected behavior**
The file names to be preserved upon uploading.

**Screenshots**
File names when uploaded in Nextcloud:
![image](https://user-images.githubusercontent.com/81660172/215295027-6a69e729-e035-46be-8404-91d318f0164e.png)

**Smartphone (please complete the following information):**
 - OS: Android 11
 - Les Pas v2.7.2

**Comment by scubajeff** [comment]
Bug confirmed. Pretty sure this is the side effect of adapting the new Android Photo Picker API. Will revert to the old implementation or try to find a way preserving the original filename.

**Comment by marcus-k** [comment]
Ah I see thanks, that's good to know. Yeah I noticed the photo picker was different whenever I updated. One down side of the new photo picker is that it does not seem to have a scroll bar to quickly drag to the end of my photos. I ended up having to spend a good minute or two scrolling all the way down to the earlier pictures I had.

I didn't know about selecting photos and sharing them to Les Pas before. I just tested that out and it worked as expected with the file names preserved. I will probably end up doing that in the meantime while this problem is being sorted out.
